.planemo.sh

```diff
@@ -9,9 +9,10 @@ echo "BLA"
 # source deactivate
 planemo database_create galaxy
 curl https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -o miniconda.sh
+chmod +x miniconda.sh
 ./miniconda.sh -b -p $tmp_dir/conda
-planemo conda_init --conda_prefix $tmp_dir/conda
-export PATH=$tmp_dir/conda/bin:$PATH
+# planemo conda_init --conda_prefix $tmp_dir/conda
```

Comment:
The mentioned planemo init is running under the given python env. Which should not work for the travis test cases in a python 3.X context. Planemo 2.7 with hicexplorer in python 3.X context is running now as it is, maybe it can be made simpler :) 
All trivial errors are gone now, but we see many non-trivial ones :(